nodes/node.go

```diff
@@ -6,6 +6,9 @@ import (
 
 // Interface for defining unit of work to be processed by event loop
 type InterfaceNode interface {
+	GetName() string
+	SetName(name string)
+	InitializeFields()
  	ProcessVirt(predecessorNodeResults []interface{})
```

Comment:
see my "runVirt()" comment for Workflow and apply the same here for `Process()` and `processVirt()`

Reply:
Done